Packs/ThreatZone/Integrations/ThreatZone/ThreatZone.py

```diff
@@ -0,0 +1,544 @@
+                           'private': result['private']}
+
+        submission_uuid = result['uuid']
+        if status == 0:
+            raise DemistoException(
+                f"Reason: {stats[status]}\nUUID: {submission_uuid}\nSuggestion: Re-analyze submission or contact us."
+            )
+
+        result_url = f"https://app.threat.zone/submission/{submission_uuid}"
+        level = result['level']
+        readable_dict = {
+            'STATUS': stats[status],
+            'MD5': md5,
+            'SHA1': sha1,
+            'SHA256': sha256,
+            'THREAT_LEVEL': levels[level],
+            'FILE_NAME': result['fileInfo']['name'],
+            'PRIVATE': result['private'],
+            'SCAN_URL': result_url,
+            'UUID': submission_uuid
+        }
+
+        output = {
+            'ThreatZone.Result(val.Job_ID == obj.Job_ID)': {
+                'STATUS': status,
+                'MD5': md5,
+                'SHA1': sha1,
+                'SHA256': sha256,
+                'LEVEL': level,
+                'INFO': submission_info,
+                'URL': result_url,
+                'UUID': submission_uuid,
+                'REPORT': {report_type: result["reports"][report_type]}
+            }
+        }
+
+        res = create_res(readable_dict, output)
+        if report_type == "cdr" and status == 5:
+            sanitized_file_url = f"https://app.threat.zone/download/v1/download/cdr/{submission_uuid}"
+            output['ThreatZone.Result(val.Job_ID == obj.Job_ID)']["SANITIZED"] = sanitized_file_url
+            readable_dict["SANITIZED"] = sanitized_file_url
+            # data = client.threatzone_get_sanitized(submission_uuid)
+            # f_res = fileResult(f"sanitized-{submission_uuid}.zip", data)
+            res = create_res(readable_dict, output)
+            # res.append(f_res)
+
+    except Exception as e:
+        output = {'ThreatZone.Result(val.Job_ID == obj.Job_ID)': {'REPORT': result}}
+        res = create_res(result, output, exception=e)
+    return res
+
+
+def threatzone_check_limits(client: Client) -> CommandResults:
+    availability = client.threatzone_check_limits(None)
+    readable_output = tableToMarkdown('LIMITS', availability["Limits"])
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_key_field='results',
+        outputs={'ThreatZone.Limits(val.Job_ID == obj.Job_ID)': availability["Limits"]}
+    )
+
+
+def threatzone_sandbox_upload_sample(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """Uploads the sample to the ThreatZone sandbox to analyse with required or optional selections."""
+    availability = client.threatzone_check_limits("sandbox")
+    if not availability["avaliable"]:
+        raise DemistoException(
+            f"Reason: {availability['Reason']}\nSuggestion: {availability['Suggestion']}\nLimits: {availability['Limits']}")
+
+    ispublic = args.get('isPublic')
```

Comment:
`isPublic` is not defined in the yml file 
and instead there is the arg `private` which are not been used in the code